### ISSUE TITLE
Macro action settings problem

### ISSUE THREAD

**Issue by tatunts** [WarmUpTill/SceneSwitcher#233]
Hello @WarmUpTill,

I've noticed a problem when you set a macro action: I've created an action that pauses another macro. If I create a new macro, change its order in the left pane macro list, if I go back to the previous macro, in the action macro pause, the previously chosen macro switches to the new one. I think it is reading the macro order instead of a "unique ID".

Thanks.

**Comment by WarmUpTill** [comment]
Hello @tatunts ,
thanks for noticing and reporting the issue!
Should be fixed with linked PR.

**Comment by tatunts** [comment]
Wonderful! I think this is also happening with `scene item visibility` action as well.

Does this PR has the latest fixes and hotkeys function?

Thanks.

**Comment by WarmUpTill** [comment]
No, the PR does not contain the changes of the other branches ;)
I have started build which does here:
https://github.com/WarmUpTill/SceneSwitcher/actions/runs/933546727

I cant reproduce the issue regarding scene item visibility - I am probably missing something obvious.
Can you give more detailed instructions?

**Comment by tatunts** [comment]
Thanks, I'll give a try.

The problem with the scene item visibility seems to be the same as the macro action: I set a scene and item, reorder macros on the left pane, close the plugin window, open again, look at the scene item visibility action and there's another item there, no the one I've previously set.

Thanks.

**Comment by WarmUpTill** [comment]
Hm, very strange. 
I have no clue why that would happen.
Can you share the settings which cause these issues with me?

The macro reference issue I can understand but I am not sure why references to OBS sources would break.
Maybe it is related to #230 which I could also not reproduce yet.

**Comment by tatunts** [comment]
No problem... Here is my latest backup:

[SS Backup.txt](https://github.com/WarmUpTill/SceneSwitcher/files/6644771/SS.Backup.txt)

I'll test this build and let you know if there are still problems.

I think it might be fixed, not sure. Looks like it just looses its reference to the previously set item source after reordering a macro.

I can imagine this is not as simple as it looks, even to create a reorder system to the trigger and action sections.

Thank you!
